.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,41 @@
+## Checklist
+- Code is compiling ✅ ❌
+- Code is prettified ✅ ❌
+- Contains unit tests ✅ ❌
+- Contains breaking changes ✅ ❌
+
+#### Web specific
+- Contains e2e tests ✅ ❌
+- Is accessible ✅ ❌
+- Compatible with Studio ✅ ❌
+- Cross-browser compatible ✅ ❌
+
+#### Native specific
+- Works in Android ✅ ❌
+- Works in iOS ✅ ❌
+- Works in Tablet ✅ ❌
+
+#### Feature specific
+- Comply with designs ✅ ❌
+- Comply with PM's requirements ✅ ❌
+
+**_Please remove unnecessary emojis and sections and this comment before proceed_**
+
+## This PR contains
+- [ ] Bug fix
+- [ ] Feature
+- [ ] Refactor
+- [ ] Documentation
+- [ ] Other (describe)
+
+## What is the purpose of this PR?
+_Please add your explanation about the purpose of the PR_
+
+## Relevant changes
+_Please add a high level explanation of what was changed and how the initial problem was solved_
+
+## How to test?
```

Comment:
I don't know what @leonardomendix thinks, I feel we shouldn't be prescriptive in detailing _how_ to test but rather what _should_ be covered in the least to ensure functionality and appearance, as well as any areas that may need extra attention.

Reply:
@leonardomendix your opinion here please

Reply:
Yes, we kind already follow these coverage idea, so maybe we can change the title to 'Test coverage' and remove the how to test in the next line to be more clear.

Reply:
Done